### PR TITLE
Box with radios

### LINKED ISSUE #263: Add an ammo box full of the radios
It would be nice to have an ammo box that is filled with all of the radios like TFAR

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Adds  a box with radios.
- Closes #263 

Future:
- Add a custom box model -> #269

### CODE REVIEW

File: addons/sys_radio/CfgVehicles.hpp [hunk 1]
```diff
@@ -1 +1,21 @@
...
+#define NUM_RADIOS_IN_CRATE 50
+
+class CfgVehicles {
+    class NATO_Box_Base;
+    class ACRE_radioSupplyCrate: NATO_Box_Base {
+        scope = 2;
+        scopeCurator = 2;
+        accuracy = 1000;
+        model = "\A3\weapons_F\AmmoBoxes\AmmoBox_F";
+        author = "ACRE2-Team";
```
Comment (jonpas) [inline addons/sys_radio/CfgVehicles.hpp]: `author = ECSTRING(main,Author);`

File: addons/sys_radio/CfgVehicles.hpp [hunk 2]
```diff
@@ -1 +1,21 @@
-//CfgVehicles.hpp
+#define NUM_RADIOS_IN_CRATE 50
+
+class CfgVehicles {
+    class NATO_Box_Base;
+    class ACRE_radioSupplyCrate: NATO_Box_Base {
+        scope = 2;
+        scopeCurator = 2;
+        accuracy = 1000;
```
Comment (jonpas) [inline addons/sys_radio/CfgVehicles.hpp]: Is that needed?

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/sys_radio/CfgVehicles.hpp (hunk 2 above)
Comment (TheMagnetar) [reply addons/sys_radio/CfgVehicles.hpp]: yes, NATO_Box_Base has nothing of this defined. It is totally empty.

**Comment by PabstMirror** [comment]
I think it needs to be in the `units[] ={}` array in cfgPatches for it to show in 3den/zeus

**Comment by jonpas** [comment]
Yes it does.

File: addons/sys_radio/stringtable.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACRE">
+    <Package name="sys_radio">
+        <Key ID="STR_ACRE_sys_radio_radioSupplyCrate">
+            <English>[ACRE2] Radio Supply Crate</English>
```
Comment (jonpas) [inline addons/sys_radio/stringtable.xml]: Not used?

**Review by jonpas — CHANGES_REQUESTED** [review]

File: addons/sys_radio/stringtable.xml (hunk 3 above)
Comment (jonpas) [reply addons/sys_radio/stringtable.xml]: Done.

**Review by jonpas — APPROVED** [review]

**Pull request merged**